polkadot-parachains/src/service.rs

```diff
@@ -1510,7 +1475,7 @@ pub fn canvas_kusama_build_import_queue(
 			Ok((timestamp, slot))
 		},
 		registry: config.prometheus_registry(),
-		can_author_with: sp_consensus::CanAuthorWithNativeVersion::new(client.executor().clone()),
+		can_author_with: sp_consensus::AlwaysCanAuthor,
```

Comment:
This was a bit weird for me, but @gilescope explained to me that the on-chain runtime is always up-to-date (hopefully), at least there is no reason why a node would assume it is out-of-date, therefore a new block can always be authored with it.

Reply:
The on-chain runtime is the canonical runtime, you don't need to hope that it is up-to-date. There is no way that this isn't up-to-date.

This `can_author_with` feature was added back in the days that when you break the authoring interface, that you have a nicer way of telling the node that it is outdated. However, we never really needed it and when the native runtime is removed it will be gone anyway. We may need to find another way or just ignore that and let the block building fail.